test/model-helpers/Profile.test.js

```diff
@@ -0,0 +1,48 @@
+import * as solidClient from '@inrupt/solid-client';
+import { beforeEach, afterEach, describe, expect, vi, it } from 'vitest';
+import { fetchProfileInfo } from '../../src/model-helpers';
+
+let session = {};
+const mockWebId = 'https://example.com/pod/profile/card#me';
+
+vi.mock('@inrupt/solid-client');
+
+describe('fetchProfileInfo', () => {
```

Comment:
Would also like some tests for `uploadProfileInfo`, 'uploadProfileImage`, and `removeProfileImage`

Reply:
Got the new unit tests for the three other functions in `Profile.test.js` included with this PR.